src/applications/accredited-representative-portal/containers/POARequestDetails.jsx

```diff
@@ -0,0 +1,173 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useParams, Link } from 'react-router-dom-v5-compat';
+import { formatDateShort } from 'platform/utilities/date/index';
+import usePOARequests from '../hooks/usePOARequests';
+
+const checkAuthorizations = (
+  isTreatmentDisclosureAuthorized,
+  isAddressChangingAuthorized,
+  status,
+) => {
+  const authorizations = [];
+  if (isTreatmentDisclosureAuthorized === status) {
+    authorizations.push('Health');
+  }
+
+  if (isAddressChangingAuthorized === status) {
+    authorizations.push('Address');
+  }
+  return authorizations.length > 0 ? authorizations.join(', ') : 'None';
+};
+
+const POARequestDetails = () => {
+  const { poaRequests } = usePOARequests();
+  const { id } = useParams();
+
+  const poaRequest =
+    poaRequests && poaRequests.find(r => r.id === Number(id))?.attributes;
+
+  return (
+    <section className="poa-request-details">
+      <h1>
+        <span data-testid="poa-request-details-header">POA request:</span>
+        {poaRequest?.claimant?.firstName} {poaRequest?.claimant?.lastName}
+      </h1>
+
+      <span
+        className="poa-request-details__divider"
+        aria-hidden="true"
+        tabIndex={-1}
+      />
+
+      <h2>Veteran information</h2>
+
+      <ul className="poa-request-details__list">
+        <li className="poa-request-details__list-item">
+          <p className="poa-request-details__title">Name</p>
+          <p className="poa-request-details__subtitle">
+            {poaRequest?.veteran?.lastName}, {poaRequest?.veteran?.firstName}
+          </p>
+        </li>
+
+        <li className="poa-request-details__list-item">
+          <p className="poa-request-details__title">VA file number</p>
+          <p className="poa-request-details__subtitle">xxx-xxx-1111</p>
+        </li>
+        <li className="poa-request-details__list-item">
+          <p className="poa-request-details__title">Social security number</p>
+          <p className="poa-request-details__subtitle">xxx-xx-1234</p>
+        </li>
+      </ul>
+
+      <h2>POA request information</h2>
+
+      <ul className="poa-request-details__list">
+        <li className="poa-request-details__list-item">
+          <p className="poa-request-details__title">POA submission date</p>
+          <p className="poa-request-details__subtitle">
+            {poaRequest?.submittedAt &&
+              formatDateShort(poaRequest?.submittedAt)}
+          </p>
+        </li>
+        <li className="poa-request-details__list-item">
+          <p className="poa-request-details__title">
+            {poaRequest?.status === 'Declined' && (
+              <va-icon
+                icon="close"
+                class="poa-request__card-icon--red poa-request__card-icon"
```

Comment:
If we're using this class more generically than for just the card, we should make the class name and where its styles are defined more generic than for just the card.